### PR TITLE
Cleanup RDSE unit tests

### LINKED ISSUE #2037: Cleanup RDSE unit tests
Part of #2000

### PR DESCRIPTION
**Pull request by oxtopus**
Fixes #2037

### CODE REVIEW

File: tests/unit/nupic/encoders/random_distributed_scalar_test.py [hunk 1]
```diff
@@ -78,8 +81,8 @@ def testEncoding(self):
     """
     # Initialize with non-default parameters and encode with a number close to
     # the offset
-    enc = RandomDistributedScalarEncoder(name='enc', resolution=1.0, w=23,
-                                         n=500, offset = 0.0)
+    enc = RandomDistributedScalarEncoder(name="enc", resolution=1.0, w=23,
```
Comment (david-ragazzi) [inline tests/unit/nupic/encoders/random_distributed_scalar_test.py]: @oxtopus How about use `encoder` instead of `enc`?

**Comment by chetan51** [comment]
:+1:

**Comment by oxtopus** [comment]
@david-ragazzi "enc" is now "encoder"

**Comment by david-ragazzi** [comment]
> @david-ragazzi "enc" is now "encoder"

Thanks, after Travis is green we merge it..

**Pull request merged**